test/iron-ajax.html

```diff
@@ -50,6 +50,17 @@
                  debounce-duration="150"></iron-ajax>
     </template>
   </test-fixture>
+  <!-- note(rictic):
+       This makes us dependent on a third-party server, but we need to be able
+       to check what headers the browser actually sends on the wire.
+       If necessary we can spin up our own httpbin server, as the code is open
+       source.
+    -->
```

Comment:
The tests below are certainly important, but the dependency on a third party server in this case is very unhygienic. We don't really have a good strategy in place to tell WCT to spin up additional processes before running tests, either. What do you think the long term plan for solving this problem should look like?